style.css

```diff
@@ -63,4 +63,372 @@
+    border-radius: 0.5rem;
+  }
+
+  #snap2 {
+    background-color: #d0c7d1;
+  }
+
+  #snap3 {
+    background-color: rgb(231, 119, 44);
+  }
+
+  #snap4 {
+    background-color: rgb(255, 255, 255);
+  }
+
+  .project_name {
+    margin-top: 0.2rem;
+    font-family: monospace;
+    font-style: normal;
+    font-weight: 1200;
+    font-size: 1.8rem;
+    line-height: 2.75rem;
+    margin-bottom: -0.5rem;
+  }
+
+  .details {
+    display: inline-flex;
+    flex-direction: row;
+  }
+
+  .detail {
+    display: inline-flex;
+    color: #344563;
+    padding-right: 2.6rem;
+    margin-left: -2.3rem;
+  }
+
+  .projo {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .info_block {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 0.75rem;
+    width: 18.438rem;
+    height: 18rem;
+  }
+
+  .info {
+    font-family: monospace;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 0.938rem;
+    line-height: 1.5rem;
+    color: #344563;
+    margin-top: -1rem;
+  }
+
+  .languages {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 0.5rem;
+    width: 13.375;
+    height: 1.5rem;
+    margin-left: -13%;
+  }
+
+  .language {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 12px;
+    width: 57px;
+    background: #ebebff;
+    border-radius: 8px;
+    height: 16px;
+    font-family: monospace;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070fd;
+  }
+
+  #html {
+    width: 2.063rem;
+  }
+
+  #css {
+    width: 1.56rem;
+  }
+
+  #javascript {
+    width: 4.25rem;
+  }
+
+  .button {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    padding: 12px;
+    gap: 10px;
+    width: 126px;
+    height: 48px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    font-family: monospace;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 15px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+  }
+
+  .button:hover {
+    background: #6070ff;
+    box-shadow: 8px 16px rgba(64, 83, 252, 0.24);
+    color: #fff;
+  }
+
+  .button:active {
+    background: #2230d2;
+    color: #fff;
+  }
+
+  .button:disabled {
+    background: #e5e5e5;
+    color: #5e6c84;
+  }
+
+  .work {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 114px 24px;
+    gap: 88px;
+    position: absolute;
+    width: 375px;
+    height: 2788px;
+    top: 718px;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 114px 24px;
+    gap: 24px;
+    position: absolute;
+    width: 375px;
+    height: 1062.15px;
+    top: 3506px;
+    margin-top: -8rem;
+    border: 0.1rem 0.1rem 0.1rem 0.1rem solid #c9cedb;
+    border-radius: 1rem;
+  }
```

Comment:
- [ ] As I mentioned previously please make sure to use the proper font for the texts in the "About" section as well as there seems to be a discrepancy in your design:
- Your design:
  ![image](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/104968236/a1359d44-53f6-4370-ade6-fe2d3fd079e6)
- Figma design:
  ![image](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/104968236/4bcfd134-695b-4fbd-8960-ce5e11283a5d)